src/testdir/test_spell.vim

```diff
@@ -999,13 +999,21 @@ func Test_spell_screendump_spellcap()
   call VerifyScreenDump(buf, 'Test_spell_3', {})
 
   " Deleting a full stop removes missing Cap in next line
-  call term_sendkeys(buf, "5Gddk$x")
+  call term_sendkeys(buf, "5Gdd\<C-L>k$x")
```

Comment:
```diff
@@ -999,13 +999,21 @@ func Test_spell_screendump_spellcap()
   call VerifyScreenDump(buf, 'Test_spell_3', {})

   " Deleting a full stop removes missing Cap in next line
-  call term_sendkeys(buf, "5Gddk$x")
+  call term_sendkeys(buf, "5Gdd\<C-L>k$x")
   call VerifyScreenDump(buf, 'Test_spell_4', {})
```
This test was ineffective because without forcing a redraw the SpellCap highlight is not there in the first place after "dd". This could/should be resolved as per discussion in https://github.com/vim/vim/issues/12420#issuecomment-1560184585. It may be worth noting that the SpellCap highlight already _is_ updated when 'conceallevel' > 0, I think we should also force redrawing the old cursorline when 'spell' is set to make sure that the spell highlighting is updated after these kinds of edits.